ext/php8/handlers_pcntl.c

```diff
@@ -0,0 +1,84 @@
+#include <php.h>
```

Comment:
The same changes to the PHP 7 version of this file apply here too. :)

Reply:
done